apps/src/gamelab/GameLabJr.interpreted.js

```diff
@@ -67,12 +67,16 @@ function removeBehavior(sprite, behavior) {
   sprite.behaviors.splice(index, 1);
 }
 
+function createBehavior(func, extraArgs) {
+  return {
+    func: func,
+    extraArgs: extraArgs,
+  };
+}
+
 function normalizeBehavior(behavior) {
   if (typeof behavior === 'function')  {
-    behavior = {
-      func: behavior,
-      extraArgs: [],
-    };
+    behavior = createBehavior(behavior, []);
```

Comment:
I don't understand the purpose of this change

Reply:
Ah, I meant to use this function in the behavior getter block, I haven't added support for parameterized behaviors to these blocks yet, so this isn't needed.

Reverted for now.